deploy

```diff
@@ -44,6 +45,13 @@ const { ECS, Credentials } = require('aws-sdk');
     const task = taskDefinitionDescription.taskDefinition;
     console.log(`Current task definition: ${task.taskDefinitionArn}`);
 
+    const containerEnvironmentVariables = (config.containerEnvPairs || [])
+      .map(keyValueString => {
+        const parts = keyValueString.split(/=(.+)/);
```

Comment:
If the value contains "=" then we end up with truncated strings. I would suggest to split only on the first occurrence of "="

Reply:
It's exactly what it does actually, I made it specifically with regex and not with `split('=')` so that it only splits on the first occurrence, not globally.

Reply:
Switched it to `indexOf` and removed the leftover `filter`

Reply:
:bow:

Reply:
Please check again, changed to the better regex.

Reply:
Beautiful